tests/testthat/test-detect.R

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Do we need two newlines here? I'm not sure if this is a convention or not.

Reply:
Nop, not a convention, mistake. Changed.